### ISSUE TITLE
Feature Request: Avatar independent of the scene.

### ISSUE THREAD

**Issue by ADimensionBeyond** [adlnet/Sandbox#250]
Is there any way to make the avatar not a scene object-like a separate entity? In OpenSim the avatar comes into a sim and is not tied to the region objects other than it’s presence there. In Virtual World Sandbox the avatar is another fixed object. When you go from one mode to another the avatar jumps back to where it initially was. When changing modes the avatar needs to stay where it was last.

**Comment by rchadwic** [comment]
Yes- I'm not entirely clear on whether or not we want the avatar to respond to the play/pause or not. In your use case, I think there is a call for a world where the game is always "playing". 

In the development branch, you'll see we've done some work on the avatars. They are still handled pretty much the same as on the client, but the server now has some special logic to persist changes to the avatar independent of the scene. We probably won't be hugely changing much in the engine, so the GUI may need to be updated to prevent some actions like selection of the avatar.

If you're interested in changing the behavior in play/pause, look in support/client/lib/view/EditorView/publisher.js
